### PR TITLE
F/wep 131 richtext table for website client

### PR DESCRIPTION
**Pull request by faebebin**
Only corrected comment typo. 'white spice' -> 'whitespace'

### CODE REVIEW

**Review by roosnic1 — APPROVED** [review]
Perfect 🎉

**Review by 0815pascal — APPROVED** [review]

**Review by penk85 — DISMISSED** [review]

**Review by bishoymelek — APPROVED** [review]

File: examples/website/src/shared/atoms/richText.tsx [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+export const tableStyle = cssRule({
+  width: '100%',
+  borderCollapse: 'collapse',
+  tableLayout: 'fixed'
+})
+
+export const tableCellStyle = cssRule({
+  border: '1px solid',
+  padding: '8px',
+  // remove extra white spice after paragraph inside of table-cell
```
[the remark this answers is not in the document]
Comment (roosnic1) [reply examples/website/src/shared/atoms/richText.tsx:15]: White spice??? is that a fancy name for flour?

[the remark this answers is not in the document]
Comment (faebebin) [reply examples/website/src/shared/atoms/richText.tsx:15]: Will correct :D. thanks

**Review by penk85 — APPROVED** [review]

**Review by roosnic1 — APPROVED** [review]

**Review by 0815pascal — APPROVED** [review]
Looks good 👍🏼

**Pull request merged**